db/migrate/20190502191756_create_targets.rb

```diff
@@ -0,0 +1,14 @@
+class CreateTargets < ActiveRecord::Migration[5.2]
+  def change
+    create_table :targets do |t|
+      t.belongs_to :topic, foreign_key: true
+      t.belongs_to :user, foreign_key: true
+      t.integer :area_length, default: 0, null: false
+      t.string :lat
```

Comment:
I think these `lat, lng` are going to be more helpful if you treat them as a float